src/script/scriptfunc.cpp

```diff
@@ -984,16 +984,30 @@ bool runRadar(CBotVar* var, std::function<bool(std::vector<ObjectType>, float, f
     {
         while ( array != nullptr )
         {
+            if (array->GetValInt() == OBJECT_MOBILEpr)
+            {
+                type_v.push_back(OBJECT_MOBILEwt);
```

Comment:
Wrong place, this should go in CObjectManager::Radar so that it also works in other functions like search() and level files

Reply:
I've been trying _stuff_ in object_manager. Just wanted to get out of there, heh. You're right, this is not a perfect solution, gotta work on that.

Reply:
Yeah, CObjectManager doesn't want to work with that. I'll just add the same code for select() and detect() - it's simpler this way, and the game won't need to go through the same vector multiple times. I think this is the better solution. Give me a minute.

Reply:
Yeah, I'm sorry about that, but I can't seem to make it work without greater changes in both scriptfunc and object_manager. We should probably rework this whole system in the future tho.

Reply:
I'm resolving this conversation since this pull request is already old and the code can be always be refactored later.